bucket/vieb.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "4.0.0",
+    "description": "Vieb is the Vim Inspired Electron Browser",
+    "homepage": "https://vieb.dev/",
+    "license": "GPL-3.0-only",
```

Comment:
invalid SPX identifier

Reply:
This is what I found on https://spdx.org/licenses/GPL-3.0-only.html. What identifier I should put there?

Reply:
Sorry my bad. I overlooked it.

Reply:
I want to mention that it should actually be https://spdx.org/licenses/GPL-3.0-or-later.html

Reply:
Updated license to GPL-3.0-or-later